internal/store/pod.go

```diff
@@ -827,6 +914,110 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_pod_container_resource_limits_cpu_cores",
+			Type: metric.Gauge,
+			Help: "The number of CPU cores requested limit by a container.",
+			GenerateFunc: wrapPodFunc(func(p *v1.Pod) *metric.Family {
+				ms := []*metric.Metric{}
+
+				for _, c := range p.Spec.Containers {
+					req := c.Resources.Limits
+
+					for resourceName, val := range req {
+						if resourceName == v1.ResourceCPU {
+							ms = append(ms, &metric.Metric{
+								LabelKeys:   []string{"container"},
+								LabelValues: []string{c.Name},
+								Value:       float64(val.MilliValue()) / 1000,
```

Comment:
Again here like above can't we just use scale value?

Reply:
Same answer. Extented resources in kubernetes, can only be [whole amounts](https://kubernetes.io/docs/tasks/administer-cluster/extended-resource-node/#discussion), which is why the scaled value is correct for all other cases, but this one, as CPU is always specify-able in millis, and just gets parsed as such. Scaled value can't represent this. The types are arguably super confusing.

Reply:
Same answer as above in the API code itself https://github.com/kubernetes/apimachinery/blob/master/pkg/api/resource/quantity.go#L702 it seems like MilliValue function uses the `q.ScaledValue(Milli)`